apps/dialtone-documentation/docs/.vuepress/theme/components/Navbar.vue

```diff
@@ -117,14 +117,131 @@
         </svg>
       </span>
     </a>
+    <dt-dropdown
+      id="theme-toggle-dropdown"
+      navigation-type="arrow-keys"
+      placement="bottom-start"
+      class="theme-toggle-dropdown"
+      max-height="33vh"
+    >
+      <template #anchor>
+        <dt-button
+          v-dt-tooltip:bottom="`Theme: ${currentTheme.charAt(0).toUpperCase() + currentTheme.slice(1)} `"
+          class="theme-toggle-button dialtone-shell-btn"
+          importance="clear"
+          kind="muted"
+        >
+          <template #icon>
+            <dt-icon
+              size="400"
+              name="satisfied-filled"
+            />
+          </template>
+        </dt-button>
+      </template>
+      <template #list>
+        <dt-list-item-group
+          heading-class="d-py4 d-px8 d-c-default d-fc-tertiary d-label--sm"
+          heading="Base Theme"
+        >
+          <dt-list-item
+            role="menuitem"
+            navigation-type="arrow-keys"
+            @click="setTheme('dp')"
+          >
+            Dialpad (DP)
+            <template #right>
+              <dt-icon :class="{ 'd-o0': currentTheme !== 'dp' }" name="check" size="200" />
+            </template>
+          </dt-list-item>
+        </dt-list-item-group>
+        <dt-dropdown-separator />
+        <dt-list-item-group
+          heading-class="d-py4 d-px8 d-c-default d-fc-tertiary d-label--sm"
+          heading="Partner Themes"
+        >
+          <dt-list-item
+            role="menuitem"
+            navigation-type="arrow-keys"
+            @click="setTheme('tmo')"
+          >
```

Comment:
Funny, this change shows a warning that is a missing declare event of `click` in `emits`.

For some reason there is a `click` event in vue2 but not in vue3.



https://github.com/user-attachments/assets/ca8265c8-7377-41f6-9820-79ea02f31ca0





please add this code (the same we have in vue2)
here: 
packages/dialtone-vue3/components/list_item/list_item.vue

```
/**
     * Native click event
     *
     * @event click
     * @type {PointerEvent | KeyboardEvent}
     */
    'click',
```

On `emits: [`